src/ElementFinderInterface.php

```diff
@@ -34,7 +34,7 @@ public function content(string $expression, bool $outerContent = false): StringC
      * ```
      *
      * @param string $expression
-     * @return $this
+     * @return ElementFinder
```

Comment:
@akravets12 why do you change the return type to ElementFinder?

Reply:
okay, its better to return ElementFinderInterface
ElementFinder is immutable and new realization of remove method returns ElementFinderInterface
```     * @param string $expression
     * @return ElementFinder
     */
    final public function remove($expression): ElementFinderInterface
    {
        return $this->modify($expression, new RemoveElements());
    }
    /**
     * @param string $expression
     * @param ElementFinderModifierInterface $modifier
     * @return ElementFinderInterface
     */
    public function modify(string $expression, ElementFinderModifierInterface $modifier): ElementFinderInterface
    {
        $result = clone $this;
        $modifier->modify(
            $result->query($expression)
        );
        return $result;
    }```

Reply:
Return type should be `$this` or `ElementFinderInterface`. Prefer `ElementFinderInterface`